scapy/automaton.py

```diff
@@ -859,11 +859,6 @@ def _do_control(self, ready, *args, **kargs):
                 self.cmdout.send(m)
             self.debug(3, "Stopping control thread (tid=%i)" % self.threadid)
             self.threadid = None
-            # Close sockets
```

Comment:
According to your comment, I assume that these lines do not work. Could you explain why?

Reply:
I added this not long ago, but it appears the sockets are always closed at this point.

Reply:
Thanks.